openapi/src/test/groovy/io/micronaut/openapi/MyJaxbElement4.java

```diff
@@ -0,0 +1,90 @@
+package io.micronaut.openapi;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+public class MyJaxbElement4 {
```

Comment:
`@Schema(oneOf = {DiscountSizeOpenApi.class, DiscountFixedOpenApi.class, MultiplierSizeOpenApi.class})` 
Test for this block. Fixed processing in this PR